src/__tests__/dependency-resolution.js

```diff
@@ -367,6 +367,22 @@ tape('dependency registration with undefined value', t => {
   t.end();
 });
 
+tape('dependency registration with null value', t => {
```

Comment:
This may be a duplicate of [this](https://github.com/fusionjs/fusion-core/blob/di/src/__tests__/dependency-resolution.js#L339), albeit without the optional token component.

Reply:
Yea, it is a slightly different case I think